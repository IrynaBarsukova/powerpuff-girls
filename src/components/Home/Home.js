import React from 'react';
import { Link } from 'react-router-dom';

import Container from '../common/Container/Container';
import styles from './Home.scss';

function Home() {
    return (
        <div className={styles.home}>
            <Container>
                <Link to="/shows/6771" className={styles.home__link}>Watch show</Link>
            </Container>
        </div>
    );
}

export default Home;
