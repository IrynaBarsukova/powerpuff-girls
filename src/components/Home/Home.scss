@import '../../styles/variables';

.home {
  padding: 20px 0;
}

.home__link {
  align-self: flex-start;
  color: white;
  padding: 10px;
  background: $pink;
  border: 2px solid $black;

  &:hover {
    background: lighten( $pink, 10% )
  }
}
