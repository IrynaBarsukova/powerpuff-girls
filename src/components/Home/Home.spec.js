import React from 'react';

import renderWithRouter from '../../mocks/router.mock';
import Home from './Home';

describe('<Home />', () => {
    it('should render Home', () => {
        const { container } = renderWithRouter(<Home />);

        expect(container).toMatchSnapshot();
    });
});
