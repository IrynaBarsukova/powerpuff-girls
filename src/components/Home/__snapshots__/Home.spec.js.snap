// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render Home 1`] = `
<div>
  <div
    class="home"
  >
    <div
      class="container"
    >
      <a
        class="home__link"
        href="/shows/6771"
      >
        Watch show
      </a>
    </div>
  </div>
</div>
`;
